src/Modules/LIKQ/FanoutSearch.UnitTest/JsonDSLTest2.cs

```diff
@@ -272,9 +272,9 @@ public void JsonDSLTest2_11()
   },
   'paper': {
     'type': 'Paper',
-    'select': [ 'OriginalTitle', 'ConferneceID' ],
+    'select': [ 'OriginalTitle', 'ConferenceID' ],
```

Comment:
I hope this is just a test that doesn't care about the spelling...

Reply:
it's got the wrong spell from the beginning so we adapted to that in the test. :/

Reply:
so please revert this one. :)